app/views/users/show.html.erb

```diff
@@ -1 +1,11 @@
-<h1>This is the user page</h1>
\ No newline at end of file
+<div class="user-page">
+  <%= render partial: "shared/user", locals: { user: @user } %>
+  <div class="user-bio">
+    <h2>Bio</h2>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <%= render partial: "shared/post", collection: @posts %>
+  
+  <%= link_to "See All Posts", "/users/#{@user.id}/posts", class: "see-posts-btn" %>
+</div>
```

Comment:
- [ ] There is an error when I navigate to user show route, this is because in your schema you have `user_id` but in your model you are referencing author, kindly be specific with the table name you create in your DB and the models too
   
![Screenshot from 2022-10-27 19-08-03](https://user-images.githubusercontent.com/66097371/198366578-eb63168a-49c8-49f7-a98a-fc185d02eaa2.png)

Reply:
@VICTOR-CHUKWU In the schema it is author_id not user_id. I have sent you a slack message if you can make this clearer for me.

Reply:
kindly delete your last migration file as it is changing author_id to user_id :+1:

Reply:
@VICTOR-CHUKWU Thanks!